doc/assessment.md

```diff
@@ -77,35 +106,133 @@ An assessment is broken down in to a list of zones, like this:
+`maxPoints` | number | The maximum points available for this question on a Homework that allows multiple attempts for more points. (Optional: default: same as `points`)
+`id` | string | The question ID if this slot contains just one question (can’t be specified with `alternatives`). (Optional; default: none)
+`alternatives` | array | The list of question alternatives if this slot contains multiple alternative questions (can’t be specified with `id`). (Optional; default: none)
+`numberChoose` | integer | If `alternatives` are specified, the number of them to select. (Optional; default `1`).
+`triesPerVariant` | integer | The maximum number of attempts allowed for each question variant (on Homeworks). (Optional; default `1`)
+`forceMaxPoints` | boolean | Whether to force all students to receive maximum points. See [Regrading](regrading.md). (Optional; default `false`)
+
+## Question alternative lists
+
+
+
+            "properties": {
+                "points": {
+                    "$ref": "#/definitions/points"
+                },
+                "maxPoints": {
+                    "$ref": "#/definitions/points"
+                },
+                "id": {
+                    "$ref": "#/definitions/questionId"
+                },
+                "forceMaxPoints": {
+                    "$ref": "#/definitions/forceMaxPoints"
+                },
+                "triesPerVariant": {
+                    "description": "The maximum number of graded submissions allowed for each question instance.",
+                    "type": "number",
+                    "default": 1
+                }
+            }
+
+FIXME
+
 
-* Within each zone the question order is randomized for `Exam` assessments.
+Once the question `id` is determined, then a random variant of that question is selected. Question alternatives inherit the points of their parent group, if specified.
 
-* An assessment question can be specified by either a single `id` or by a list of alternatives, in which case one or more of these alternatives is chosen at random. Once the question `id` is determined, then a random variant of that question is selected. Question alternatives inherit the points of their parent group, if specified.
+## Assessment instances, question selection, and question order
 
-* If a zone has `maxPoints`, then, of the points that are awarded for answering questions in this zone, at most `maxPoints` will count toward the total points.
+PrairieLearn distinguishes between *assessments* and *assessment instances*. An *assessment* is determined by the code in an `assessments` directory, and is something like "Midterm 1". Given an assessment, PrairieLearn needs to select the random list of questions for each student, and it is this selection that is the *assessment instance* for the student. There is only one copy of each assessment, but every student has their own assessment instance.
```

Comment:
How do you feel about adding a *slash* to emphasize directories? e.g. `assessments/`

Reply:
good idea, done